index.html

```diff
@@ -681,8 +681,8 @@ <h4 id="sanitization">Sanitization</h4>
         <dt><code>allow="media"</code></dt>
         <dd>
           This allows the rendering of <code>&lt;img></code>,
-          <code>&lt;picture></code>, <code>&lt;video></code> 
-          <code>&lt;audio></code>, and <code>&lt;object></code> elements in the
+          <code>&lt;picture></code>, <code>&lt;video></code>,
+          <code>&lt;audio></code> and <code>&lt;object></code> elements in the
```

Comment:
I prefer oxford commas if we could please.
```suggestion
          <code>&lt;audio></code>, and <code>&lt;object></code> elements in the
```
But this line will also need wrapping I think?

Reply:
> I prefer oxford commas if we could please.

Sure, I was not actually aware of the Oxford comma until now 🙂